src/components/Tags/Tags.tsx

```diff
@@ -16,7 +33,7 @@ const Tags = () => {
       setTags([...MOCK_TAGS])
       return
     }
-    const newTags = MOCK_TAGS.filter(tag => tag.includes(value))
+    const newTags = MOCK_TAGS.filter(tag => tag.includes(value.toLowerCase()))
```

Comment:
Shouldn't there be a `.toLowerCase` before `includes`, like:  `tag.toLowerCase().includes(value.toLowerCase())`?